_posts/2017/2017-01-17-testing-async-interactions.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: post
+title: "How to test event-based services using contracts"
```

Comment:
I am not sure - was just joking around - we already had this title for Oli's post last year[ "How to make microservice testing great again"](https://developer.epages.com//blog/2016/07/14/wiremock.html). Isn't this kind of phrase ("...great again") a little overused. But these are just my doubts - you decide